cmake/extensions.cmake

```diff
@@ -229,14 +229,14 @@ Please provide one of following: CONF_FILES")
 
   if(NOT DEFINED PREPROCESS_ARGS_BOARD)
     # Defaulting to system wide settings when BOARD is not given as argument
-    set(board_in ${BOARD})
+    set(board_combined ${BOARD})
```

Comment:
why not do a proper cleanup instead ? ref: https://github.com/gmarull/sdk-nrf/pull/13#discussion_r1570646695